tap_github/schemas/issue_events.json

```diff
@@ -657,817 +187,800 @@
+                  "type": ["null", "string"]
+                },
+                "allow_squash_merge": {
+                  "type": ["null", "boolean"]
+                },
+                "allow_auto_merge": {
+                  "type": ["null", "boolean"]
+                },
+                "delete_branch_on_merge": {
+                  "type": ["null", "boolean"]
+                },
+                "allow_update_branch": {
+                  "type": ["null", "boolean"]
+                },
+                "use_squash_pr_title_as_default": {
+                  "type": ["null", "boolean"]
+                },
+                "allow_merge_commit": {
+                  "type": ["null", "boolean"]
+                },
+                "subscribers_count": {
+                  "type": ["null", "integer"]
+                },
+                "network_count": {
+                  "type": ["null", "integer"]
+                }
+              }
+            },
+            "temp_clone_token": {
+              "type": ["null", "string"]
+            },
+            "allow_squash_merge": {
+              "type": ["null", "boolean"]
+            },
+            "allow_auto_merge": {
+              "type": ["null", "boolean"]
+            },
+            "delete_branch_on_merge": {
+              "type": ["null", "boolean"]
+            },
+            "allow_update_branch": {
+              "type": ["null", "boolean"]
+            },
+            "use_squash_pr_title_as_default": {
+              "type": ["null", "boolean"]
+            },
+            "allow_merge_commit": {
+              "type": ["null", "boolean"]
+            },
+            "allow_forking": {
+              "type": ["null", "boolean"]
+            },
+            "subscribers_count": {
+              "type": ["null", "integer"]
+            },
+            "network_count": {
+              "type": ["null", "integer"]
+            },
+            "open_issues": {
+              "type": ["null", "integer"]
+            },
+            "watchers": {
+              "type": ["null", "integer"]
+            },
+            "master_branch": {
+              "type": ["null", "string"]
+            },
+            "starred_at": {
+              "type": ["null", "string"]
+            }
+          }
         },
-        "following_url": {
+        "assignees": {
           "type": [
             "null",
-            "string"
-          ]
+            "array"
+          ],
+          "items": {
+            "$ref": "shared/user.json#/"
+          }
         }
       }
     },
-    "review_requester": {
+    "id": {
+      "type": ["null", "integer"]
+    },
+    "actor": {
+      "$ref": "shared/user.json#/"
+    },
+    "node_id": {
+      "type": ["null", "string"]
+    },
+    "performed_via_github_app": {
+      "$ref": "shared/performed_via_github_app.json#/"
+    },
+    "_sdc_repository": {
+      "type": ["null", "string"]
+    },
+    "rename": {
       "type": [
```

Comment:
Can you please keep the type values in one line like below to maintain the consistency.
      "type": ["null", "object"],

Reply:
Updated to keep it as a single line